src/ray/common/cgroup/README.md

```diff
@@ -0,0 +1,35 @@
+### Ray core cgroup documentation
+
+#### Physical execution mode
+
+Ray core supports a physical execution mode, which allows users to cap resource consumption for their applications.
+
+A few benefits:
+- It prevents application from eating up unlimited resource to starve other applications running on the same node;
+- It protects other processes on the node (including ray system components) to be killed due to insufficient resource (i.e. OOM);
+
+TODO(hjiang): reserve minimum resource will be supported in the future.
+
+#### Disclaimer and presumption
```

Comment:
It's not a disclaimer. What about: "Prerequisites"

Reply:
Updated.